dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheHashMap.java

```diff
@@ -876,8 +1094,18 @@ private void processInvalidListeners() {
      */
     @Override
     protected BackedSession readFromExternal(String id) {
+        final boolean trace = TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled();
```

Comment:
As in my second comment, another spot where a different value for trace is used.

Reply:
updated